plugins/woocommerce-admin/client/marketplace/components/tabs/tabs.scss

```diff
@@ -45,6 +45,18 @@
 		text-align: center;
 		z-index: 26;
 	}
+
+	&__update-count-extensions {
+		background-color: #000;
+	}
+
+	&__update-count-themes {
+		background-color: #000;
+	}
+
+	&__update-count-business-services {
+		background-color: #000;
+	}
```

Comment:
How about this or just using the same class name for all those elements?
```suggestion
	&__update-count-extensions,
	&__update-count-themes,
	&__update-count-business-services {
		background-color: #000;
	}
```

Reply:
I see that there's a class `woocommerce-marketplace__update-count` already so it could be just

```css
&__update-count {
	background-color: #000;
}
```

Reply:
Also, based on the designs I see that only active tab should have dark bg. Inactive tabs should have darker font and lighter background.

Reply:
> Also, based on the designs I see that only active tab should have dark bg. Inactive tabs should have darker font and lighter background.

This is dependent on the behavior you noticed (search cleared when clicking on a tab). I will note it down and address it in that other issue.

Reply:
CSS styling unified in ca443b7fda